### PR TITLE
Dev chain fast

### PR DESCRIPTION
**Pull request by samt1803**

### CODE REVIEW

File: packages/dev-chain-fast/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "builda": "docker build -t streamr/dev-chain-fast .",
+    "buildb": "./prepare.sh",
+    "deploy": "tsx deploy.ts",
+    "clean": "rm -fr node_modules"
+  },
+  "author": "Streamr Network AG",
+  "license": "ISC",
+  "dependencies": {
+    "@streamr/hub-contracts": "file:../hub-contracts",
+    "@streamr/network-contracts": "file:../network-contracts",
```
Comment (jtakalai) [inline packages/dev-chain-fast/package.json:17]: Why is this needed? What does it do? Shouldn't monorepo take care of this?

**Review by jtakalai — COMMENTED** [review]
After adding this to npm workspaces, `npm run build` in the root failed for me during the `docker build` step at `npm i`. Maybe it's not supposed to be part of it? Should it be located elsewhere than under packages?

File: packages/config/src/networks.json [hunk 2]
```diff
@@ -71,6 +71,37 @@
...
+    "dev2": {
+        "id": 31337,
+        "rpcEndpoints": [
+            {
+                "url": "http://10.200.10.1:8547"
+            }
+        ],
+        "adminPrivateKey": "0x5e98cce00cff5dea6b454889f359a4ec06b9fa6b88e9d69b86de8e1c81887da0",
+        "contracts": {
+            "ENS": "0x3AE0ad89b0e094fD09428589849C161f0F7f4E6A",
```
Comment (jtakalai) [inline packages/config/src/networks.json:83]: needs to be updated I guess?

File: packages/dev-chain-fast/prepare.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+npm run clean
```
Comment (jtakalai) [inline packages/dev-chain-fast/prepare.sh:1]: maybe first delete the tgz files just in case they exist

File: packages/dev-chain-fast/prepare.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+cd ../..
+npm run build -w @streamr/network-contracts
+npm run build -w @streamr/hub-contracts
+cd packages/network-contracts
+npm pack
+cd ../hub-contracts
+npm pack
+cd ../dev-chain-fast
+npm i ../network-contracts/streamr-network-contracts-*.tgz
+npm i ../hub-contracts/streamr-hub-contracts-*.tgz
```
Comment (jtakalai) [inline packages/dev-chain-fast/prepare.sh:11]: delete the tgz files after

File: packages/hub-contracts/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@streamr/hub-contracts",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
[the remark this answers is not in the document]
Comment (jtakalai) [reply packages/hub-contracts/package.json:3]: so after merging:
* bump version
* publish
* commit, tag & push

File: packages/network-contracts/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@streamr/network-contracts",
-  "version": "4.2.0",
+  "version": "4.3.0",
```
Comment (jtakalai) [inline packages/network-contracts/package.json:3]: bump version when publishing

**Review by jtakalai — APPROVED** [review]

File: packages/dev-chain-fast/package.json (hunk 1 above)
Comment (samt1803) [reply packages/dev-chain-fast/package.json:18]: this will be changed in a flollowing ticket, that makes the image smaller
